src/main/java/bookmarked/command/AddCommand.java

```diff
@@ -34,8 +35,8 @@ public void processAddCommand(String[] newSplitBook, ArrayList<Book> listOfBooks
         if (newSplitBook.length < 1 || newSplitBook[1].isBlank()) {
             throw new emptyArgumentsException();
         }
-
-        this.listOfBooks.add(new Book(newSplitBook[1].trim()));
-        System.out.println("Added!");
+        Book bookName = new Book(newSplitBook[1].trim());
+        this.listOfBooks.add(bookName);
```

Comment:
nice that you factor out `new Book ......` into another variable first before placing it as a parameter to a method, making it more readable